### PR TITLE
OpenRGB 0.8

### PR DESCRIPTION
**Pull request by DarthGandalf**
Based on #28694 by @scardracs 

From the 3 packaged plugins, app-misc/openrgb-plugin-effects-0.8 has new dependencies: https://github.com/justxi/QCodeEditor (fork of a fork of a fork...) and https://github.com/SRombauts/SimplexNoise so I skipped it for now, and only updated the other 2 plugins.

### CODE REVIEW

File: dev-cpp/cpp-httplib/cpp-httplib-0.11.3.ebuild
Comment (AndrewAmmerlaan) [inline dev-cpp/cpp-httplib/cpp-httplib-0.11.3.ebuild:6]: It doesn't build with 3.8 or 3.9?

Comment (DarthGandalf) [reply dev-cpp/cpp-httplib/cpp-httplib-0.11.3.ebuild]: It does, added. Do we need to care about old python for new packages?

Comment (AndrewAmmerlaan) [reply dev-cpp/cpp-httplib/cpp-httplib-0.11.3.ebuild]: > It does, added. Do we need to care about old python for new packages?

Well it probably doesn't really matter for this package, since the python dependency is build-time only. But we usually drop a python implementation throughout the entire tree in one go to avoid making a dependency mess. Older systems could still be using 3.9, we still support this.

**Comment by xgqt** [comment]
Hi!
I wanted to check out how OpenRGB 0.8 plays with OpenRazer and because of this bumped `openrazer` _locally_.

There are two noticeable changes from the one I made:
1. openrgb insists on adding `-save-temps` during build
   whcich causes C++ compiler warnings, so filter `-pipe`
   ```bash
        # > warning: ‘-pipe’ ignored because ‘-save-temps’ specified
        filter-flags -pipe
   ```
2. udev rules were getting installed into `/usr`,
   I had removed them after installing using the udev eclass
   ```bash
        udev_dorules 60-openrgb.rules
        rm -r "${ED}"/usr/lib/udev/
   ```

**Comment by DarthGandalf** [comment]
@xgqt thanks, updated. Do you want to add yourself only to openrgb itself, or also to plugins, or also to the new dependencies to co-maintain?

**Comment by xgqt** [comment]
> @xgqt thanks, updated. Do you want to add yourself only to openrgb itself, or also to plugins, or also to the new dependencies to co-maintain?

I can only test with Razer devices now, so right now I will add myself to `app-misc/openrgb` only, but that I will do after this is merged.

Speaking of merging, I think the current developer-maintainer @chutz should check this out.

**Review by chutz — APPROVED** [review]

**Comment by chutz** [comment]
Changes approved from the OpenRGB side, as to the new dependencies, i will leave that to the proxy-maint team for now.

**Comment by xgqt** [comment]
@DarthGandalf @scardracs @chutz Thanks a lot!

File: app-misc/openrgb/openrgb-0.8.ebuild
Comment (AndrewAmmerlaan) [inline app-misc/openrgb/openrgb-0.8.ebuild:74]: Btw, what is happening here? I had to remove this line in order to have openrgb detect some `/dev/hidraw` devices. Why do we install this udev file only to delete it again?

**Comment by xgqt** [comment]
so, your udev dir is in /usr/lib? on my openrc system the files are installed into /lib/udev/rules.d/.

**Comment by AndrewAmmerlaan** [comment]
> so, your udev dir is in /usr/lib? on my openrc system the files are installed into /lib/udev/rules.d/.

I'm on the `default/linux/amd64/17.1/desktop/plasma/systemd/merged-usr` profile, so `/usr/lib/` and `/lib/` are the same thing on my system. If the intent here is to remove the rules from `/usr/lib` and install them to `/lib`, might I suggest changing the order of these two lines, i.e. remove first and then add. That should fix the installation of the udev rules on merged-usr systems.

**Comment by xgqt** [comment]
yea, exactly, the order here is wrong anyway

**Comment by scardracs** [comment]
> > so, your udev dir is in /usr/lib? on my openrc system the files are installed into /lib/udev/rules.d/.
> 
> 
> 
> I'm on the `default/linux/amd64/17.1/desktop/plasma/systemd/merged-usr` profile, so `/usr/lib/` and `/lib/` are the same thing on my system. If the intent here is to remove the rules from `/usr/lib` and install them to `/lib`, might I suggest changing the order of these two lines, i.e. remove first and then add. That should fix the installation of the udev rules on merged-usr systems.
> 
> 
> 
> 

That explains why on my installation no devices were found at all (same profile)

**Comment by scardracs** [comment]
@AndrewAmmerlaan could you quick fix it?

**Comment by xgqt** [comment]
already pushed a revbump

**Pull request closed without merging**
